core/object/message_queue.h

```diff
@@ -33,15 +33,15 @@
 
 #include "core/object/object_id.h"
 #include "core/os/thread_safe.h"
+#include "core/templates/local_vector.h"
 #include "core/variant/variant.h"
-
```

Comment:
Not sure if you remove those by habit or if your IDE does this but it's really nicer looking if `#include`s are in one block separated from `class` definitions.

Reply:
QtCreator clang-format support does this (and other weird stuff) sometimes. I generally catch it doing it but often I entirely miss it. We should try to have more CI or precommit hooks somehow to catch these things (as in also spaces between functions), should not be super difficult with a python script if we ask for volunteers.